mne/io/curry/curry.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: UTF-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+import mne
+import os
+import re
+
+
+def _read_curry(full_filename):
+    '''
+    Read curry into python.
+    Inspired by Matt Pontifex' EEGlab loadcurry() extension
+
+    :param full_filename:
+    :return:
+    '''
+
+    # we don't use os.path.splitext to also handle extensions like .cdt.dpa
+    file, ext = full_filename.split(".", maxsplit=1)
+
+    if 'cdt' in ext:
+        curry_vers = 8
+        if not (os.path.isfile(file + ".cdt") and
+                os.path.isfile(file + ".cdt.dpa")):
+            raise FileNotFoundError("The requested filename %s does not have "
+                                    "both file components (.cdt, .cdt.dpa) "
+                                    "created by Curry 8." % file)
```

Comment:
Just say which of the files missing right?

Reply:
Sure, just was bluntly recreating the EEGLAB loadcurry solution, but yeah, it would probably be better to tell which one is missing.